### PR TITLE
Summer new intern proposal

### PR DESCRIPTION
**Pull request by ojas1**
Added description for new interns

### CODE REVIEW

File: src/summer-internship-position-2020/index.org [hunk 1]
```diff
@@ -171,3 +171,46 @@ projects.
...
+   *Project Description*
+
+   Virtual Labs Engineering and Architecture Division (VLEAD) is
+   looking for competent and experienced application developer to work
+   on the experiment development framework for Virtual Labs. You will
+   be working on developing an architecture for building virtual labs
+   experiments and a set of tools to help developers build better
+   experiments. These tools in the form of libraries, will be a part
+   of an experiment development framework, which is targeted towards
+   delivering experiments in the form of salable web applications. You
```
Comment (travula) [inline src/summer-internship-position-2020/index.org:189]: scalable

File: src/summer-internship-position-2020/index.org [hunk 2]
```diff
@@ -171,3 +171,46 @@ projects.
...
+   will also be developing some computer science experiments using the
+   same tools we build.  As a part of this project, you will be
```
Comment (travula) [inline src/summer-internship-position-2020/index.org:191]: As part of

File: src/summer-internship-position-2020/index.org (hunk 1 above)
Comment (ojas1) [reply src/summer-internship-position-2020/index.org]: updated

File: src/summer-internship-position-2020/index.org (hunk 2 above)
Comment (ojas1) [reply src/summer-internship-position-2020/index.org]: updated

**Pull request merged**